src/reps/grip.js

```diff
@@ -96,7 +96,7 @@ const GripRep = React.createClass({
     }
 
     const truncate = Object.keys(properties).length > max;
-    let props = this.getProps(properties, indexes, truncate);
+    let props = this.getProps(object.class === "Proxy", properties, indexes, truncate);
```

Comment:
I think this deserves to be extracted to a `const suppressQuotes = object.class === "Proxy";` together with a comment explaining why we don't need to quote property names for a proxy.